File: src/main/java/me/miki/shindo/management/event/impl/EventRender2D.java
package me.miki.shindo.management.event.impl;

import me.miki.shindo.management.event.Event;

public class EventRender2D extends Event {

    private final float partialTicks;

    public EventRender2D(float partialTicks) {
        this.partialTicks = partialTicks;
    }

    public float getPartialTicks() {
        return partialTicks;
    }
}

File: src/main/kotlin/com/shindoclient/shindo/management/event/impl/EventRender2D.kt
package com.shindoclient.shindo.management.event.impl

import com.shindoclient.shindo.management.event.Event

class EventRender2D(
    @JvmField val partialTicks: Float,
) : Event() {
    fun getPartialTicks(): Float = partialTicks
}
